pom.xml

```diff
@@ -585,6 +586,21 @@
 		</dependency>
 	</dependencies>
 
+	<pluginRepositories>
+		<pluginRepository>
+			<releases>
+				<enabled>false</enabled>
+			</releases>
+			<snapshots>
+				<enabled>true</enabled>
+			</snapshots>
+
+			<id>sonatype-nexus-snapshots</id>
```

Comment:
If we can remove the SNAPSHOT dependency, we can probably remove this repository as well.